indexer/locals.tf

```diff
@@ -19,7 +19,7 @@ locals {
   }
 
   service_secret_ids = {
-    for name in local.service_names : name => "${var.environment}-${name}-secrets"
+    for name in local.service_names : name => "${var.aws_db_secret_id}"
```

Comment:
Was this change made so that a deployer could determine what secret to create in secrets manager before deploying?
There's other secrets that exist today, this could break deployments that had diff. secret ids right?

Reply:
Good point, addressed.

Reply:
Right now, there's existing dev-vulcan-secrets/dev-ender-secrets/etc. We'll need to add db_password into these secrets manually. Env variables are encrypted & passed into lambdas.